windows-msi/build.wsf

```diff
@@ -329,28 +329,6 @@ clean	Cleans intermediate and output files</example>
                     break;
             }
 
-            // Add rules for 7-Zip SFX creation.
```

Comment:
There's a *case "exe"* just above this (lines 326-329). Should it be removed also?

Reply:
Removed more stuff :)